web/react/utils/utils.jsx

```diff
@@ -124,8 +124,10 @@ module.exports.notifyMe = function(title, body, channel) {
 }
 
 module.exports.ding = function() {
-  var audio = new Audio('/static/images/ding.mp3');
-  audio.play();
+    if (!navigator || !navigator.userAgent || navigator.userAgent.toLowerCase().indexOf("firefox") === -1) {
```

Comment:
Yeah, if it's being used in two places like this it should be a single utils function